### ISSUE TITLE
@JuliaRegistrator bot doesn't distinguish code block and normal paragraph

### ISSUE THREAD

**Issue by tkf** [JuliaRegistries/Registrator.jl#240]
I tried to "comment out" pinging JuliaRegistrator bot by putting it in a code block (so that I can copy-and-paste later)

>     @JuliaRegistrator `register(branch=release)`

but JuliaRegistrator actually interpreted it as the `register` command.  See: https://github.com/tkf/Transducers.jl/issues/69

It was surprising to me since I naively assumed that it works by receiving the @-notification.  But it is rather minor glitch so I'm not sure if it is worth fixing.

**Comment by DilumAluthge** [comment]
Yeah even though it has the `@`-syntax, it's not actually responding to a username ping. Registrator is installed as a GitHub app that receives webhook events for every issue comment event, PR comment event, and commit comment event.

It is probably a good idea to require that the registration comment occurs at the beginning of a line.
